lib/core/feature-flags/src/lib/services/storage-features.service.ts

```diff
@@ -0,0 +1,140 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Inject, Injectable, Optional } from '@angular/core';
+import { BehaviorSubject, Observable, of } from 'rxjs';
+import { map, skip } from 'rxjs/operators';
+import {
+    FlagChangeset,
+    IFeaturesService,
+    FlagSet,
+    IWritableFeaturesService,
+    WritableFeaturesServiceConfigToken,
+    WritableFlagChangeset,
+    WritableFeaturesServiceConfig
+} from '../interfaces/features.interface';
+import { FlagSetParser } from './flagset.parser';
+import { StorageService } from 'public-api';
```

Comment:
This seems wrong?

Reply:
I have to choose from
`'public-api'` ❔ 
`'../../../..'` points to `lib/core/index` ❔ 
absolute paths ❌ 

In `lib/core/shell/src/lib/shell.module.ts` (inside core but not touched by this PR) we have this import
```
import { SidenavLayoutModule } from '@alfresco/adf-core';
```
I did not use this import because lint is complaining about it
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/36277451/a3b55f0a-99ce-4142-bcad-cec4ae10fd7b)
Should I use this type of import beside ts complains? Should instead this import be updated with some of the above or another one (which?) import? @DenysVuika

Reply:
updated import with `'../../../..'`,
'public-api' make unit test fail

Reply:
Not sure how it is with subpackages of ngpackagr, but if this work in the shell submodule, maybe this is the way? Regarding absolute imports you are right, it doesn't seem right, unless ngpackgr can handle the build. Can it?